__tests__/app.test.js

```diff
@@ -76,4 +76,26 @@ describe('GET /api/articles/:article_id', () => {
             expect(response.body.msg).toBe('bad request')
         })
     })
+})
+
+describe('GET /api/articles', () => {
+    test('returns a 200 status code and an array of articles sorted by date in descending order', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.articles).toHaveLength(13)
+            expect(body.articles).toBeSorted({ descending: true })
+            body.articles.forEach((article) => {
+                expect(typeof article.author).toBe('string')
+                expect(typeof article.title).toBe('string')
+                expect(typeof article.article_id).toBe('number')
+                expect(typeof article.topic).toBe('string')
+                expect(typeof article.created_at).toBe('string')
+                expect(typeof article.votes).toBe('number')
+                expect(typeof article.article_img_url).toBe('string')
+                expect(typeof article.comment_count).toBe('number')
+            })
+        })
```

Comment:
Should add a test to make sure the returned articles don't have a body property 🧪